src/pages/api/howru.js

```diff
@@ -3,23 +3,29 @@ import * as workFragments from "@/lib/api/work.fragments";
 import * as searchFragments from "@/lib/api/search.fragments";
 import { getServerSession } from "@dbcdk/login-nextjs/server";
 
+import { getErrorCount, resetErrorCount } from "@/utils/errorCount";
+
 const upSince = new Date();
 
 let session;
+const maxError500 = 10;
 
 /**
  * The howru handler
  *
  * We make requests for all our GraphQL fragments
  */
 export default async function handler(req, res) {
-  if (!session) {
-    session = await getServerSession(req, res);
-  }
-
   // If any of the services fail, this is set to false
   let ok = true;
 
+  try {
+    if (!session) {
+      session = await getServerSession(req, res);
+    }
+  } catch (e) {
+    console.log(e);
```

Comment:
dette bliver ikke logget i json-format som vi har brug for. Der skal bruges dbc logger biblioteket